plugins/net_plugin/net_plugin.cpp

```diff
@@ -2725,8 +2738,9 @@ namespace eosio {
             if( ec ) {
                fc_wlog( logger, "Addresses fetching ticked sooner than expected: ${m}", ("m", ec.message()) );
             }
-            for( auto& c : connections ) {
-               if (c->connected() && c->protocol_version >= proto_address_advertising) {
+            for (auto itr = connections.lower_bound(connection_state::connected); itr != connections.end(); ++itr) {
```

Comment:
Where is checking on `connected` ?

Reply:
@afalaleev All of them are connected because `is_open() is false` = `close() not called` = `connection_state::none` which is before `connection_state::connected` in the index..